webpack.common.js

```diff
@@ -97,6 +97,9 @@ const config = [
 			'data-stores': [
 				assets + 'data/index.js',
 			],
+			'eventespresso-core-blocks': [
+				assets + 'blocks/index.js',
+			],
```

Comment:
I don't think this should be in this branch since there is no `assets/src/blocks/index.js` in this branch?

Reply:
Ya so this is what is causing the fail on travis because the `npm build` step in the job fails with this entry.  So removing it will fix that.